criu/proc_parse.c

```diff
@@ -853,6 +853,7 @@ int parse_pid_stat(pid_t pid, struct proc_pid_stat *s)
 		return -1;
 	}
 
+	buf[n] = 0;
```

Comment:
buf is always null-terminated

Reply:
Ah, because of the struct member end which comes after buf.